CardPayments/src/main/java/com/paypal/android/cardpayments/CardClient.kt

```diff
@@ -92,7 +92,11 @@ class CardClient internal constructor(
                 )
 
                 if (response.payerActionHref == null) {
-                    val result = CardResult(response.orderId)
+                    val result = CardResult(
+                        orderId = response.orderId,
+                        status = response.status?.name,
```

Comment:
What is the .name Steven? I thought response.status was a String?

Reply:
Yeah the `name` property is built-in to [Kotlin enums](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-enum/name.html).